Airline_Web_API/Airline_Web_API/Services/TicketTypeService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Airline_Web_API.Helpers;
+using Airline_Web_API.Models;
+using Airline_Web_API.ViewModels;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Airline_Web_API.Services
+{
+    public class TicketTypeService
+    {
+        private readonly AirlineContext _context;
+        private readonly IMapper _mapper;
+
+        public TicketTypeService(AirlineContext airlineContext, IMapper mapper)
+        {
+            _context = airlineContext;
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<TicketTypeViewModel>> GetTicketTypesAsync()
```

Comment:
Why this method returns IEnumerable but the same method in TicketsService returns List? Which is correct?